utils/src/unit-test/java/uk/gov/moj/sdt/utils/AbstractSdtXmlTestBase.java

```diff
@@ -250,7 +250,7 @@ public void error(final SAXParseException e) throws SAXException {
                     expectedMessages.remove(e.getMessage());
                 } else {
                     LOGGER.error("Parser encountered unexpected error: [" + e.getMessage() + "]");
```

Comment:
Should the LOGGER.error message use the {} place holder instead of string concatenation?

Reply:
LOGGER removed altogether